docs/proposal-component-names.md

```diff
@@ -0,0 +1,60 @@
+# Proposal (Feb 2020)
+
+## Cylc 8 System Component Naming
+
+Cylc is now a complex multi-component system. It is important that we settle on
+consistent, unambiguous, and where possible standard, component names so that
+users and developers can communicate without confusion. (This was not easy even
+before Cylc 8!).
+
+## Proposed Final Names
+
+Note we have already agreed to drop **suite** for **workflow** ("suite" is
+non-standard outside of the NWP community).
+
+| Component Name               | Code Repository   | Config File                                          | Description |
+| --                           |----               | ---                                                  |---          |
+|:boom: **Cylc Scheduler**     | cylc-flow         | :boom: `cylc-flow.conf` and `cylc-flow-global.conf`  | an instance of the cylc-flow scheduler program, to manage a single workflow | 
+|**Cylc Hub**                  | (Jupyter Hub)     | :boom: `<cylc-location>/jupyterhub_config.py`               | the single point of access that users log in to |
+|**Cylc UI Server**            | cylc-uiserver     | N/A? (or :boom: `cylc-uiserver.conf`)                | an instance of the cylc-uiserver program, that serves the Cylc Web UI and CLI |
```

Comment:
```diff
- `cylc-uiserver.conf`
+ `uiserver.py` ?
```

Reply:
Again, only if we "cylc" in a directory hierachy that the file is required to reside in.

Reply:
Which is the plan?

Reply:
Like the cylc-flow global file, not decided yet as far as I'm aware.